### ISSUE TITLE
Newest openssl (openssl-1.0.2g.tgz) breaks python

### ISSUE THREAD

**Issue by plluksie** [joyent/smartos-live#579]
After updating to the openssl-1.0.2g python throws errors:

```
# pip reinstall virtualenv
Traceback (most recent call last):
  File "/opt/local/bin/pip2.7", line 9, in <module>
    load_entry_point('pip==7.1.2', 'console_scripts', 'pip2.7')()
  File "/opt/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 568, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/opt/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2720, in load_entry_point
    return ep.load()
  File "/opt/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2380, in load
    return self.resolve()
  File "/opt/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2386, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/opt/local/lib/python2.7/site-packages/pip/__init__.py", line 15, in <module>
    from pip.vcs import git, mercurial, subversion, bazaar  # noqa
  File "/opt/local/lib/python2.7/site-packages/pip/vcs/subversion.py", line 9, in <module>
    from pip.index import Link
  File "/opt/local/lib/python2.7/site-packages/pip/index.py", line 30, in <module>
    from pip.wheel import Wheel, wheel_ext
  File "/opt/local/lib/python2.7/site-packages/pip/wheel.py", line 35, in <module>
    from pip._vendor.distlib.scripts import ScriptMaker
  File "/opt/local/lib/python2.7/site-packages/pip/_vendor/distlib/scripts.py", line 14, in <module>
    from .compat import sysconfig, detect_encoding, ZipFile
  File "/opt/local/lib/python2.7/site-packages/pip/_vendor/distlib/compat.py", line 31, in <module>
    from urllib2 import (Request, urlopen, URLError, HTTPError,
ImportError: cannot import name HTTPSHandler
```

downgrade to openssl-1.0.2f.tgz solves the issue

**Comment by rmustacc** [comment]
Is this in reference to a specific pkgsrc repository? Can you include what image version you're using?

@jperkin does this look familiar?

**Comment by plluksie** [comment]
Instance (base-64-lts 15.4.0)
http://pkgsrc.joyent.com/packages/SmartOS/2015Q4/x86_64/All

**Comment by askfongjojo** [comment]
@plluksie - Please refer to this issue https://github.com/joyent/pkgsrc/issues/326. I don't believe there is anything we can fix. It's a dependency of python 2.7 itself on SSLv2 which has to be disabled for security reason. (https://www.openssl.org/news/openssl-1.0.2-notes.html)

**Comment by plluksie** [comment]
@askfongjojo @jperkin @rmustacc thank you. Enforcing the reinstall with the newest packages solved the issue:

```
pkgin -fy up
pkgin -y upgrade
pkgin -Fy in python27
```

**Comment by jperkin** [comment]
This will be fixed correctly in the 15.4.1 images which will be released shortly.
